### PR TITLE
Adds the borb plushie !

### PR DESCRIPTION
**Pull request by LeChatGuide**
## About The Pull Request

Adds an adorable borb plushie !

borb

## Why It's Good For The Game

Because we always need more plushies , and this is bird station anyway .

## Changelog
:cl:
add: Added the borb plushie 
imageadd: added the plushie_borb icon 
/:cl:

### CODE REVIEW

File: code/game/objects/items/plushes.dm [hunk 1]
```diff
@@ -521,6 +521,15 @@
 	attack_verb_simple = list("bite", "hiss", "tail slap")
 	squeak_override = list('sound/weapons/bite.ogg' = 1)
 
+/obj/item/toy/plush/borbplushie
+	name = "borb plushie"
+	desc = "An adorable stuffed toy that resembles a round , fluffy looking bird. Not to be mistaken for his friend , the birb plushie."
```
Comment (Avunia) [inline code/game/objects/items/plushes.dm:526]:
```suggestion
	desc = "An adorable stuffed toy that resembles a round, fluffy looking bird. Not to be mistaken for his friend, the birb plushie."
```

**Comment by Avunia** [comment]
Also your changelog is a bit borked.

```md
## Changelog
:cl:
add: Added the borb plushie
imageadd: added the plushie_borb icon 
/:cl:
```
This should be enough. Images won't work ingame and soundadd is for *new* sounds.
If you wish you can also add a custom name to be displayed by putting it after the first `:cl:`
for example
```md
:cl: LeChatGuide
```

**Comment by LeChatGuide** [comment]
alright, I modified the description and the changelog this should be fixed now.

**Comment by Avunia** [comment]
Went ahead and moved the image also out of the changelog. Code itself is fine. c:

**Review by Avunia — APPROVED** [review]

**Pull request merged**